server/channels/app/platform/license.go

```diff
@@ -126,11 +126,19 @@ func (ps *PlatformService) SaveLicense(licenseBytes []byte) (*model.License, *mo
 		return nil, model.NewAppError("addLicense", "api.unmarshal_error", nil, "", http.StatusInternalServerError).Wrap(jsonErr)
 	}
 
+	if license.Features == nil {
+		return nil, model.NewAppError("addLicense", "api.license.add_license.invalid.app_error", nil, "", http.StatusBadRequest)
+	}
+
 	uniqueUserCount, err := ps.Store.User().Count(model.UserCountOptions{})
 	if err != nil {
 		return nil, model.NewAppError("addLicense", "api.license.add_license.invalid_count.app_error", nil, "", http.StatusBadRequest).Wrap(err)
 	}
 
+	if license.Features.Users == nil {
```

Comment:
Could move this before the db call to save the call when nil.

Reply:
Good catch, thanks! Done in https://github.com/mattermost/mattermost/pull/29417/commits/c548847acdb9ac4cafb33a6210ac751688e88dd0